js/reference.js

```diff
@@ -2769,189 +2828,60 @@ var ERMrest = (function(module) {
      * @memberof ERMrest
      * @constructor
      * @param {ERMrest.Reference} reference column's reference
-     * @param {?ERMrest.Column} base The column that this reference-column will be created based on.
-     * @param {?Object} kwargs if it's not empty then the column is being created based on foreignkey or key.
+     * @param {ERMrest.Column[]} baseCols List of columns that this reference-column will be created based on.
      * @desc
      * Constructor for ReferenceColumn. This class is a wrapper for {@link ERMrest.Column}.
      */
-    function ReferenceColumn(reference, base, kwargs) {
-
+    function ReferenceColumn(reference, cols) {
```

Comment:
I thought a `ReferenceColumn` was supposed to be for a single column, not for a set of columns. Unless I need to continue reading to understand the new structure.

Is `cols` an array of columns, but is usually just one? As in, the way you are using the `ReferenceColumn` constructor is by passing in an array of 1 column...? As I look at it more, it looks like `_simple` addresses what I'm asking about. That tells you whether it's composed of more than one column (I think that's how I'm understanding it).